api-tests/src/test/resources/config/sampleReport.yml

```diff
@@ -0,0 +1,34 @@
+key: "sampledataexport"
+uuid: "9e7dc296-2aad-11e3-a840-5b9e0b589afb"
+name: "sample.export.name"
+description: "sample.export.description"
+parameters:    # should be able to parse comments!
+  - key: "startDate"
+    type: "date"
+    label: "startDate.label"
+  - key: "endDate"
+    type: "date"
+    label: "endDate.label"
+datasets:
+  - key: "orders"
+    type: "sql"
+    config: "orders.sql"
+  - key: "encounters"
+    type: "sql"
+    config: "encounters.sql"
+designs:
+  - type: "csv"
+    properties:
+      "characterEncoding": "ISO-8859-1"
+      "blacklistRegex": "[^\\p{InBasicLatin}\\p{L}]"
+  - type: "excel"
+    template: "ExcelTemplate"
```

Comment:
Is this meant to refer to an actual file?

Reply:
Yes.  That bit is just used to test the YAML parsing, but it should really be "ExcelTemplate.xls" to make it clear, will correct.